### PR TITLE
Add EC21 to supported modems

### PR DESCRIPTION
**Pull request by DomAmato**

### CODE REVIEW

File: Hologram/Network/Modem/EC21.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+            except:
+                # This is some sort of binary data that can't be decoded so just
+                # return the bytes. We might want to make this happen via parameter
+                # in the future so it is more deterministic
+                self.logger.debug('Could not decode recieved data')
+
+            return resp
+
+    def is_registered(self):
+        return self.check_registered('+CREG') or self.check_registered('+CGREG')
```
Comment (rbalik) [inline Hologram/Network/Modem/EC21.py]: Should we do CEREG since it's LTE?

File: Hologram/Network/Modem/EC21.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                                        chatscript_file=chatscript_file, event=event)
+        self._at_sockets_available = True
+        self.urc_response = ''
+
+    def connect(self, timeout=DEFAULT_EC21_TIMEOUT):
+
+        success = super().connect(timeout)
+
+        # put serial mode on other port
+        # if success is True:
```
Comment (rbalik) [inline Hologram/Network/Modem/EC21.py]: can we just get rid of the commented out stuff?

File: Hologram/Network/Modem/EC21.py (hunk 1 above)
Comment (DomAmato) [reply Hologram/Network/Modem/EC21.py]: yep

File: Hologram/Network/Modem/EC21.py (hunk 2 above)
Comment (DomAmato) [reply Hologram/Network/Modem/EC21.py]: how many AT ports does the EC21 have? we could uncomment it and switch AT ports if possible

Comment (rbalik) [reply Hologram/Network/Modem/EC21.py]: I think just one. The other one seems to be a qualcomm debug port

**Review by rbalik — APPROVED** [review]

**Review by rbalik — APPROVED** [review]

**Pull request merged**